Dangerfile.swift

```diff
@@ -432,7 +432,7 @@ func checkDescriptionSection() {
 
 func commentDescriptionSection(desc: String) {
     let count = desc.trimmingCharacters(in: .whitespacesAndNewlines).count
-    if count == 0 {
+    if count == 0 { // swiftlint:disable:this empty_count
```

Comment:
I didn't add this, this was stopping me from pushing my code.
Surprised no one else is blocked by it?

Rule does not make sense here because count is an `Int`